package.json

```diff
@@ -23,25 +23,33 @@
     "main": "./src/module/fluid.js",
     "scripts": {
         "prepublish": "grunt buildDists",
-        "test": "node tests/node-tests/basic-node-tests.js"
+        "pretest": "node node_modules/rimraf/bin.js reports/* coverage/*",
+        "test": "npm run test:browser && npm run test:node",
+        "test:browser": "node node_modules/testem/testem.js ci --file tests/testem.js",
+        "test:node": "node node_modules/istanbul/lib/cli.js cover tests/node-tests/basic-node-tests.js --report none --print none",
```

Comment:
Thanks for taking the chance to automate the node tests as well. This is great and helpful!